### ISSUE TITLE
Next Unread should stay in list / Smart Feeds

### ISSUE THREAD

**Issue by peternlewis** [brentsimmons/NetNewsWire#560]
When selecting Next Unread, I think it would be better if it cycled through **either** real feeds or **smart feeds**, but not both.

Currently the behaviour (assuming you have everything marked read and then come back, and finished on say the TidBITs feed) you select Next Unread and it goes to the Today Smart Feed.  To me, this is a bit weird, I would prefer, since I am on a feed, instead it goes to the next unread in the next actual feed, cycling back to the first feed with an unread item instead of the first smart feed with an unread item.

Basically, I want to stay within feeds and never want to read from Smart Feeds.  The alternative is each time I start reading I will have to select the first unread feed first.

I trust that is clear what I am getting at?

**Comment by blackketter** [comment]
I came to file this issue too, but want to stay in the Smart Feed.

It seems strange that when I reach the end of the All Unread smart feed and Clean Up, pressing space or N to go to the next unread item jumps down to the first item in the first alphabetic folder.

I feel like I shouldn't need to keep jumping to back to the smart folder manually.
